core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -1698,46 +1698,46 @@ void skip0() {
 
 
                     /*
-                    * Make a sysctl() call to get the raw argument space of the
-                        * process.  The layout is documented in start.s, which is part
```

Comment:
yes the whole project config seems to be broken considering spotless, check, git, being out of sync as this is the prove of it.